test/check-application

```diff
@@ -1036,11 +1036,7 @@ class TestApplication(testlib.MachineCase):
         execute = self.execute
 
         def prepare():
-            # Create and start registry containers
-            self.execute(True, f"podman run -d -p 5000:5000 --name registry --stop-timeout 0 {IMG_REGISTRY}")
-            self.execute(True, f"podman run -d -p 6000:5000 --name registry_alt --stop-timeout 0 {IMG_REGISTRY}")
-            # Add local insecure registry into registries conf
-            self.machine.write("/etc/containers/registries.conf", REGISTRIES_CONF)
+            self.setupRegistry()
             self.execute(True, "systemctl stop podman.service")
```

Comment:
This is already included in `setupRegistry` and doesn't need to be repeated here.

Reply:
Thanks for spotting! I meant to do that, but just forgot.